browser-extension/src/popup/styles.css

```
@@ -619,3 +658,36 @@ body {
 .options-form textarea { min-height: 80px; resize: vertical; }
 
 .options-actions { margin-top: 20px; display: flex; gap: 10px; flex-wrap: wrap; }
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)